ICAFIX/ReApplyFixPipeline.sh

```diff
@@ -295,451 +127,314 @@ have_hand_reclassification()
 	fi
 }
 
-function interpret_as_bool()
-{
-    case $(echo "$1" | tr '[:upper:]' '[:lower:]') in
-    (true | yes | 1)
-        echo 1
-        ;;
-    (false | no | none | 0)
-        echo 0
-        ;;
-    (*)
-        log_Err_Abort "error: '$1' is not valid for this argument, please use TRUE or FALSE"
-        ;;
-    esac
-}
-
-# ------------------------------------------------------------------------------
-#  Main processing of script.
-# ------------------------------------------------------------------------------
-
-main()
-{
-	log_Msg "Starting main functionality"
-
-	# Retrieve positional parameters
-	local StudyFolder="${1}"
-	local Subject="${2}"
-	local fMRIName="${3}"
-	local HighPass="${4}"
+log_Msg "Starting main functionality"
 
-	local RegName
-	if [ -z "${5}" ]; then
-		RegName=${G_DEFAULT_REG_NAME}
-	else
-		RegName="${5}"
-	fi
-	
-	local LowResMesh
-	if [ -z "${6}" ]; then
-		LowResMesh=${G_DEFAULT_LOW_RES_MESH}
-	else
-		LowResMesh="${6}"
-	fi
-	
-	local MatlabRunMode
-	if [ -z "${7}" ]; then
-		MatlabRunMode=${G_DEFAULT_MATLAB_RUN_MODE}
-	else
-		MatlabRunMode="${7}"
-	fi
+# Naming Conventions and other variables
+local Caret7_Command="${CARET7DIR}/wb_command"
```

Comment:
Didn't these `local`s throw an error?

Reply:
I guess not? I did run reapply fix to completion. I will go ahead and remove them however.

Reply:
Done